code/modules/antagonists/changeling/powers/tiny_prick.dm

```diff
@@ -50,9 +50,6 @@
 		return
 	if(!AStar(user, target.loc, /turf/proc/Distance, changeling.sting_range, simulated_only = 0))
 		return
-	if(target.mind && target.mind.has_antag_datum(/datum/antagonist/changeling))
-		sting_feedback(user, target)
-		changeling.chem_charges -= chemical_cost //??

```

Comment:
only this line needs to be removed i think

Reply:
It will still show the message twice to the stinging and the stung, sting_feedback is called in another file